deployments/cloudfoundry/buildpack/README.md

```diff
@@ -0,0 +1,177 @@
+as a sidecar for the given app that's being deployed. The Collector is able to observe the app as a 
+[nozzle](https://docs.pivotal.io/tiledev/2-10/nozzle.html#nozzle) to
+the [Loggregator Firehose](https://docs.cloudfoundry.org/loggregator/architecture.html).
+The Loggregator Firehose is one of the architectures Cloud Foundry
+uses to emit logs and metrics. This means that the Splunk OpenTelemetry Collector will be observing all
+apps and deployments that emit metrics and logs to the Loggregator Firehose as long as it's running.
+
+## Installation
+- Clone this repository.
+- Change to this directory.
+- Run the following command:
+```sh
+# Add buildpack for Splunk OpenTelemetry Collector
+$ cf create-buildpack otel_collector_buildpack . 99 --enable
+```
+
+### Dependencies in Cloud Foundry Environment
+
+- `wget`
+- `jq`
+
+### Using PCF Buildpack With an Application
+This section covers basic cf CLI (Cloud Foundry Command Line Interface) commands to use the buildpack. 
+```sh
+# Basic setup, see the configuration section for more envvars that can be set
+$ cf set-env <app-name> OTEL_CONFIG <config_file_name>
+$ cf set-env <app-name> OTELCOL <desired_collector_executable_name>
+
+$ cd <my-application-directory>/
+
+# How to run an application without a manifest.yml file:
+# Note: This will provide the Collector for the app, but the Collector will not be running.
+$ cf push <app-name> -b otel_collector_buildpack -b <main_buildpack>
+
+# How to run an application with a manifest.yml file:
+# Option 1)
+$ cf push
+# Option 2)
+$ cf push <app-name> -b otel_collector_buildpack -b <main_buildpack> -f manifest.yml
+```
+
+Note: This buildpack requires another buildpack to be supplied after it, it is not allowed to
+be the last one for an app. Also, the manifest.yml file will need to provide the
+command to run the Splunk OpenTelemetry Collector as a sidecar for the application.
+
+## Configuration
+
+The following only applies if you are using the `otelconfig.yaml` config
+provided by the buildpack.  If you provide a custom configuration file for the Splunk OpenTelemetry Collector
+in your application (and refer to it in the sidecar configuration), these might not
+work unless you have preserved the references to the environment variables in the config file.
+For proper functionality, the `OTEL_CONFIG` environment variable must point to
+the configuration file, whether using the default or a customer version.
+
+Set the following environment variables with `cf set-env` as applicable to configure this buildpack, or
+include them in the `manifest.yml` file, as shown in the [included example](#sidecar-configuration).
+
+Required:
+- `RLP_GATEWAY_ENDPOINT` - The URL of the RLP gateway that acts as the proxy for the firehose,
+    e.g. `https://log-stream.sys.<TAS environment name>.cf-app.com`
+- `UAA_ENDPOINT` - The URL of UAA provider,
+    e.g. `https://uaa.sys.<TAS environment name>.cf-app.com`
```

Comment:
@crobert-1 Can you update https://github.com/signalfx/splunk-otel-collector/blob/main/.github/workflows/lychee.yml#L24 to exclude `*.cf-app.com`?

Reply:
Will do, thanks for pointing this out!